### PR TITLE
Feature: Percentage based vouching

### PR DESCRIPTION
**Pull request by Caranell**
Issue: #268 

Backend PR: https://github.com/coordinape/coordinape-backend/pull/67

### CODE REVIEW

**Review by f1lander — APPROVED** [review]
LGTM

**Review by f1lander — APPROVED** [review]
LGTM

File: src/pages/VouchingPage/NewNominationModal.tsx [hunk 1]
```diff
@@ -43,21 +43,35 @@ export const NewNominationModal = ({
...
+      } ${
+        (circle.nomination_days_limit || 0) > 1 ? 'days' : 'day'
+      }. If a nomination does not receive required percentage of votes in that period, the nomination fails`;
+    }
+
+    return `The ${circle.name} Circle requires ${
+      circle.min_vouches
+    } people total to vouch for a new member and nominations are live for ${
+      circle.nomination_days_limit
+    } ${(circle.nomination_days_limit || 0) > 1 ? 'days' : 'day'}. ${
```
Comment (f1lander) [inline src/pages/VouchingPage/NewNominationModal.tsx:65]: Can pull out all the conditional to separate variable to do not nest all of them into a string literal? and there is no needed to put `|| 0` if you cast to `Number(circle.nomination_days_limit)` atomically if the value is `undefined or null` value would be a `NaN`

`const nominationdaysLimit = Number(circle.nomination_days_limit) > 1 ? 'days' : 'day'`

**Review by f1lander — CHANGES_REQUESTED** [review]

**Review by f1lander — APPROVED** [review]
LGTM

**Review by f1lander — APPROVED** [review]
LGTM

File: src/components/FormRadioGroup/FormRadioGroup.tsx [hunk 2]
```diff
@@ -40,7 +44,7 @@ export const FormRadioGroup = ({
   };
 
   return (
-    <div>
+    <div className={classes.centered}>
```
Comment (f1lander) [inline src/components/FormRadioGroup/FormRadioGroup.tsx:47]: Meanwhile we do not decide to change out style base tools can we keep using the components that material-ui offer to us? for example, this is equivalent to use `Box` thats creates a `div` by default but an be overried if you wish to another semantic component like `span`, more info here: https://react-hook-form.com/api/useformstate/

`import { Box } from '@material-ui/core`

`<Box justifyContent="center">`

**Review by exrhizo — CHANGES_REQUESTED** [review]
I am putting this with request changes because we can't merge this until the backend change is merged.

Appreciate you looking at this.

**Comment by crabsinger** [comment]
I'll close this PR until someone wants to look into it more or I hear back from @Caranell .

**Pull request closed without merging**